release-please-config.json

```diff
@@ -15,7 +15,52 @@
   "extra-files": [
     {
       "type": "json",
-      "path": "./packages/**/package.json",
+      "path": "./packages/mgt-components/package.json",
```

Comment:
Missing an entry for the `./packages/mgt/package.json` file

Reply:
I've added this